src/k8s/selc_configmaps.tf

```diff
@@ -142,6 +142,7 @@ resource "kubernetes_config_map" "uservice-attribute-registry-management" {
   data = merge({
     APPLICATIONINSIGHTS_ROLE_NAME = "uservice-attribute-registry-management"
     POSTGRES_SCHEMA               = "attribute_registry"
+    WELL_KNOWN_URL                = format("%s/.well-known/jwks.json", var.cdn_storage_url)
```

Comment:
why not frontend url? for example selfcare.pagopa.it?

Reply:
I used the storage url in order to point directly to the latest file, even during the rotation of the keys (the cdn wil be purged, but this operation will not be immediate)